packages/features/portfolio/src/containers/portfolio-holdings/use-earnings-breakdown.tsx

```diff
@@ -201,11 +248,25 @@ export function useEarningsBreakdown(isGrouped: boolean) {
           },
           isDebt,
           tokenId: b.tokenId,
-          incentivesEarnings: totalIncentiveEarnings
-            ? totalIncentiveEarnings
-                .toFiat(baseCurrency)
-                .toDisplayStringWithSymbol(2, true, false, 'en-US', true)
-            : '0',
+          // TODO: use formatCryptoWithFiat once data is ready
+          incentivesEarnings: {
+            data: [
+              {
+                displayValue:
+                  totalIncentiveEarnings &&
+                  totalIncentiveEarnings.toFloat() > minEarningsValue
+                    ? totalIncentiveEarnings
+                        ?.toFiat(baseCurrency)
+                        .toDisplayStringWithSymbol(2)
+                    : '-',
```

Comment:
We should show zero on null

Reply:
Teddy made a ticket to always show a dash https://app.asana.com/0/1205751432906245/1207540343338899